### DISCUSSION TITLE
已经付款支付2年期费用，但账号管理处未更新

### DISCUSSION THREAD

**Discussion by Matthew21ipc** [getlantern/lantern-forum-cn#188] [Q&A]
692278694@qq.com账号已于2月25日付款支付2年期费用，但账号管理处未更新

**Comment by MaxMKMax** [comment]
@Matthew21ipc 您好，谢谢您的反馈。蓝灯已经在正视这个问题了。 我们正在修复问题和进行优化。请您耐心等待。非常抱歉影响了您的体验感。谢谢您的等待与支持。

您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

1. go to Support (转到支持)
2. select Report an Issue (选择报告问题)
3. choose issue type (选择问题类型)
4. include additional details (包括额外的详细信息)
5. hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队.

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。
我们将非常感谢您。

**Reply by tangzijiang1** [reply]
麻烦看下我的帖子，我购买了2年专业版，还是免费版

**Comment by MaxMKMax** [comment]
@Matthew21ipc  您好. 我们已经帮您查过了. 您已经是专业版, 只是您没有登陆进去您的专业版账号. 登录进去专业版账号就能使用了! 谢谢您!
![Uploading Screenshot 2024-02-27 at 9.16.39 AM (1).png…]()

**Comment by tangzijiang1** [comment]
***@***.***，同样的问题



---原始邮件---
发件人: "Mike ***@***.***&gt;
发送时间: 2024年2月28日(周三) 晚上11:22
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [getlantern/lantern-forum-cn] 已经付款支付2年期费用，但账号管理处未更新 (Discussion #188)




 
@Matthew21ipc  您好. 我们已经帮您查过了. 您已经是专业版, 只是您没有登陆进去您的专业版账号. 登录进去专业版账号就能使用了! 谢谢您!
